cockatrice/src/setsmodel.cpp

```diff
@@ -132,6 +132,12 @@ void SetsModel::toggleRow(int row, bool enable)
     emit dataChanged(index(row, EnabledCol), index(row, EnabledCol));
 }
 
+bool SetsModel::getToggleStatus(int row)
+{
+    CardSet *temp = sets.at(row);
+    return enabledSets.contains(temp) ? true : false;
```

Comment:
Just a minor remark: [QSet::contains()](http://doc.qt.io/qt-5/qset.html#contains) already returns a `bool`, so the `? true : false` part is not needed.